### ISSUE TITLE
Integration with rocketchat 

### ISSUE THREAD

**Issue by enggjiten** [samtecspg/articulate#649]
### Feature Request
> I could not find the rocketchat integration as bot.
It would be great, if articulate can work as back end bot and front end chat agent communicate to user.
I am not sure if it is in product road map ?

**Comment by dcalvom** [comment]
Hi @enggjiten sorry for the late response. We have on the road map the release of "Ubiquity". This is how we called the feature to create communication channels between articulate and messaging platforms.

We can consider Rocketchat as a possible channel, although, I had to be honest that we didn't have in our list of channels yet. Also, whenever we have this feature ready, you're more than welcome in helping us create the channel for Rocketchat in case we weren't able to find some time to implement it.
